datahub-web-react/src/app/entity/shared/tabs/Dataset/Schema/__tests__/filterSchemaRows.test.ts

```diff
@@ -35,6 +39,52 @@ describe('filterSchemaRows', () => {
         expect(expandedRowsFromFilter).toMatchObject(new Set());
     });
 
+    it('should properly filter schema rows based on description', () => {
+        const filterText = 'test';
```

Comment:
"test" here will actually also match on the fieldPath. can you make this more specific to test that we're only matching on `description`? so you could just change this to `const filterText = 'testing description';`